pkg/parser/uri_test.go

```diff
@@ -0,0 +1,58 @@
+package parser
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+var parseTests = []struct {
+	dsn      string
+	scheme   string
+	path     string
+	rawQuery string
+}{
+	{"file:///some/path/to/file.csv?format=csv", "file", "/some/path/to/file.csv", "format=csv"},
+	{"file:///some/path/to/file.csv", "file", "/some/path/to/file.csv", ""},
+	{"os:stdout?format=csv", "os", "stdout", "format=csv"},
```

Comment:
Please add a postgres dsn here as well if you want to use it for a uri :smile:

Reply:
Done